php/libraries/LorisForm.class.inc

```diff
@@ -672,7 +672,7 @@ class LorisForm
 
         // Apply filters that should be applied on all fields
         if (isset($this->filters['__ALL__'])) {
-            foreach ($this->filters['__ALL__'] as $filterType) {
+            for ($i = 0; $i < count($this->filters['__ALL__']); $i++) {
```

Comment:
filterType wasn't used, so this is converted to a for loop because you can't foreach without a key _or_ a value.

The whole logic of this loop is convoluted and I'm not sure it should be a loop at all, but this is the minimal change to get phpmd not to complain about `$filterType` being unused.